storage/cloud.py

```diff
@@ -1032,6 +1034,14 @@ def get_direct_download_url(
                 path, request_ip, expires_in, requires_cors, head, **kwargs
             )
 
+        if kwargs:
+            namespace = kwargs.get("namespace")
+            if namespace is not None:
```

Comment:
this check can be removed, the next check also makes sure that if `namespace` is `None` it doesn't go in the if